java/org/apache/coyote/http11/Http11InputBuffer.java

```diff
@@ -998,7 +998,7 @@ private HeaderParseStatus parseHeader() throws IOException {
 
             byte peek = byteBuffer.get(byteBuffer.position());
             if (headerParsePos == HeaderParsePosition.HEADER_MULTI_LINE) {
-                if ((peek != Constants.SP) && (peek != Constants.HT)) {
+                if (!(peek == Constants.SP || peek == Constants.HT)) {
```

Comment:
Removing the unnecessary brackets is fine. Applying De Morgan's law is not. Yes, the result is the same but the intention is less obvious than with the original form so please keep it.